README.md

```diff
@@ -126,6 +126,54 @@ import { CORE_FEATURES, FEATURE_KEYS } from "@components/Sidebar";
     />
 }
 ```
+## Example:
+### Add a custom plugin
+There is an example Customer plugin inside [examples](./src/examples/Customers/) folder. We're going to disable the default customer feature and replace with this custom plugin.
+
+Import the plugin from examples folder, then define the route object and sidebar in [routes](./src/routes.tsx).
+```javascript
+// routes.tsx
+
+// ... import statements
+import SupportAgentIcon from "@mui/icons-material/SupportAgent";
+import { CORE_FEATURES, FEATURE_KEYS, ItemProps } from "@components/Sidebar";
+
+const Customers = lazy(() => import("./examples/Customers"));
+
+export const routes: RouteObject[] = [
```

Comment:
It's not clear to me from this code snippet exactly what I supposed to do there? Need more step by step instructions with more context. Can this routes file also be more friendly to making this change? Is there any way to automate it similar how we do for the API? 

The target for this would be people who are not familiar with the app